Ansible/group_vars/za-chpc

```diff
@@ -0,0 +1,112 @@
+---
+yaim_path: /opt/glite/yaim
+bdii_port: 2170
+vo_sw_dir: /opt/exp_soft
+cvmfs_repos:
+  - sagrid.
+  - atlas
+vos:
+  - dteam
+  - ops
+  - sagrid.ac.za
+  - eumed
+queues:
+  - alice
+  - atlas
+  - ops
+  - sagrid
+# See https://wiki.egi.eu/wiki/MAN01_How_to_publish_Site_Information
+# use the site name as registered in GOCDB (use the same case)
+site_name: ZA-CHPC
+# contact details
+site_email: smasoka@csir.co.za
+site_security_email: smasoka@csir.co.za
+site_support_email: smasoka@csir.co.za
+# location
+site_latitude: -33.9552
+site_longitude: 18.4748
+site_description: "Centre for High-Performance Computing"
+site_other_grid: "EGI"
+site_bdii_host: "grid-bdii.chpc.ac.za"
+site_location: "Cape Town, South Africa"
+site_website: "http://www.chpc.ac.za"
+site_domain: chpc.ac.za
+
+# services at the site
+bdii_host: grid-bdii.chpc.co.za
+#apel_mysql_host: ce01-node001.c4.csir.co.za
+#px_host: myproxy.cern.ch
+#wms_host: wms.hellasgrid.gr
+# check GOCDB for relevant services in the AfricaArabia https://goc.egi.eu/portal/index.php?Page_Type=Service_Group&id=926
+#wms_host: wms.c4.csir.co.za
```

Comment:
The core services should be included in the UI - they will need to be configured. You should have WMS and Top-bdii set.